tfhe/benches/utilities.rs

```diff
@@ -383,6 +383,29 @@ impl EnvConfig {
     }
 }
 
+/// Generate a number of threads to use to saturate current machine for throughput measurements.
+pub fn throughput_num_threads() -> u64 {
+    let num_threads = rayon::current_num_threads() as f64;
+    // Add 20% more to maximum threads available.
+    (num_threads + (num_threads * 0.2)) as u64
+}
+
+pub enum BenchmarkType {
+    Latency,
+    Throughput,
+}
+impl BenchmarkType {
+    pub fn from_env() -> Result<Self, String> {
+        let raw_value = env::var("__TFHE_RS_BENCH_TYPE")
+            .map_err(|_| "no benchmark type provided in environment".to_string())?;
+        match raw_value.to_lowercase().as_str() {
+            "latency" => Ok(BenchmarkType::Latency),
+            "throughput" => Ok(BenchmarkType::Throughput),
+            _ => Err(format!("benchmark type '{raw_value}' is not supported")),
+        }
```

Comment:
no way to do both ?

Reply:
Nope, unless we want to somewhat duplicate calls to `write_to_json()` currently located at the end of the benchmark function.

Reply:
Or it would implies a third `match` arm, with a duplicate of the current two `BenchmarkType` variants.

Reply:
After second thought, it's not relevant to be able to do both latency and throughput.
Moreover it would duplicate a lot of code, hence increasing source of error for maintainers.